### DISCUSSION TITLE
Location of file that stores the URL names with “firestick” slingerplayer

### DISCUSSION THREAD

**Discussion by blasco403** [GerryDazoo/SlingerPlayer#23] [General]
Hello,

Is there a file that is created like on the windows version that specifies the list of URL's?

I would like to change the listing order without deleting and recreating in the application.

Thanks.

**Comment by GerryDazoo** [comment]
Yes, it’s called Sites.dat. But you might need “root” permissions to edit it.

 

From: blasco403 ***@***.***> 
Sent: Saturday, August 12, 2023 4:24 PM
To: GerryDazoo/SlingerPlayer ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GerryDazoo/SlingerPlayer] Location of file that stores the URL names with “firestick” slingerplayer (Discussion #23)

 

Hello,

Is there a file that is created like on the windows version that specifies the list of URL's?

I would like to change the listing order without deleting and recreating in the application.

Thanks.

**Reply by blasco403** [reply]
Thanks Gerry, I guess I will do it manually as rooting a firestick is a PITA.
